src/api/.rubocop.yml

```diff
@@ -105,6 +105,10 @@ Style/TernaryParentheses:
 Style/MethodCallParentheses:
   Enabled: true
 
+# Check for if and case statements where each branch is used for assignment
```

Comment:
And we should order this file alphabetically again!! :sob: (although unrelated to the PR)